scripts/localize_podfile.swift

```diff
@@ -26,7 +26,7 @@ let podfile = CommandLine.arguments[1]
 // versions when they're dependencies of other requested local pods.
 let implicitPods = ["FirebaseCore", "FirebaseInstanceID", "FirebaseInstallations", "Firebase",
                     "GoogleDataTransport", "GoogleDataTransportCCTSupport", "GoogleUtilities",
-                    "FirebaseCoreDiagnostics", "FirebaseRemoteConfig"]
+                    "FirebaseCoreDiagnostics", "FirebaseRemoteConfig", "FirebaseAuthInterop"]
```

Comment:
Thanks! All the interop pods should be added, but that can be done in a subsequent PR.